330 charge category credit refund (#355)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/skupriceid.md

A SKU Price ID is a unique identifier that defines the unit price used to calculate the charge. SKU Price ID can be referenced on a [*price list*](#glossary:price-list) published by a provider to look up detailed information, including a corresponding list unit price. The composition of the properties associated with the SKU Price ID may differ across providers. SKU Price ID is commonly used for analyzing cost based on pricing properties such as Terms and Tiers.

> The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage" and [ChargeClass](#chargeclass) is "Regular". The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is “Credit” or “Adjustment” and the charge is associated with a singular SkuPriceId. SkuPriceId MUST NOT be null when [ChargeClass](#chargeclass) is "Correction" and the correction is associated with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.

Reviewer: ```suggestion
The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published *price list*. A charge MUST be associated with a singular SkuPriceId when [ChargeCategory](#chargecategory) is "Purchase" or "Usage" and [ChargeClass](#chargeclass) is "Regular".  The SkuPriceId SHOULD NOT be null when the charge is associated with a singular SkuPriceId and it MUST be null when the ChargeCategory is "Tax". A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
```
I assume we should emphasize that it MUST be null in case of Tax and the rest is merely a suggestion. Moreover, further small adjustments/rephrasing expected as part of an upcoming alignment with the FOCUS feature levels PR.

Author: Hey Irina, This has been incorporated in the manual merge. Resolving

> The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage" and [ChargeClass](#chargeclass) is "Regular". The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is “Credit” or “Adjustment” and the charge is associated with a singular SkuPriceId. SkuPriceId MUST NOT be null when [ChargeClass](#chargeclass) is "Correction" and the correction is associated with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.

Reviewer 2: ```suggestion
The SkuPriceId column MUST be present in the billing data. This column MUST be of type String. SkuPriceId MUST define a single unit price used for calculating the charge. The [ListUnitPrice](#listunitprice) MUST be associated with the SkuPriceId in the provider published *price list*. The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is "Purchase" or "Usage" and [ChargeClass](#chargeclass) is "Regular". The SkuPriceId MUST NOT be null when [ChargeCategory](#chargecategory) is “Credit” or “Adjustment” and the charge is associated with a specific SkuPriceId. SkuPriceId MUST NOT be null when [ChargeClass](#chargeclass) is "Correction" and the correction is associated with a specific SkuPriceId. A given value of SkuPriceId MUST be associated with one and only one [SkuId](#skuid), except in cases of commitment discount flexibility.
```
